Content.Client/PDA/PDAMenu.xaml.cs

```diff
@@ -101,7 +110,31 @@ public void UpdateState(PDAUpdateState state)
                 IdInfoLabel.SetMarkup(Loc.GetString("comp-pda-ui-blank"));
             }
 
-            StationNameLabel.SetMarkup(Loc.GetString("comp-pda-ui-station", ("Station",state.StationName ?? Loc.GetString("comp-pda-ui-unknown"))));
+            StationNameLabel.SetMarkup(Loc.GetString("comp-pda-ui-station",
+                ("Station",state.StationName ?? Loc.GetString("comp-pda-ui-unknown"))));
+
+            var stationTime = _gameTiming.CurTime.Subtract(_gameTicker.RoundStartTimeSpan);
+
+            StationTimeLabel.SetMarkup(Loc.GetString("comp-pda-ui-station-time",
+                ("time", stationTime.ToString("hh\\:mm\\:ss"))));
+
+            if (state.StationAlert.Level != null)
+            {
+                StationAlertLevelInstructions.SetMarkup(Loc.GetString("comp-pda-ui-station-alert-level-instructions",
+                        ("AlertLevelInstructions", Loc.GetString($"alert-level-{state.StationAlert.Level}-instructions"))));
+                StationAlertLevelLabel.SetMarkup(Loc.GetString("comp-pda-ui-station-alert-level",
+                    ("ColorLevel", state.StationAlert.Color),
+                    ("AlertLevel", Loc.GetString($"alert-level-{state.StationAlert.Level}"))));
+            }
+            else
+            {
+                StationAlertLevelLabel.SetMarkup(Loc.GetString("comp-pda-ui-station-alert-level",
+                    ("ColorLevel", "white"), ("AlertLevel", Loc.GetString("comp-pda-ui-unknown"))));
+                StationAlertLevelInstructions.SetMarkup(
+                    Loc.GetString("comp-pda-ui-station-alert-level-instructions",
+                    ("AlertLevelInstructions", Loc.GetString("comp-pda-ui-unknown"))));
+            }
```

Comment:
You should just use ternary operators instead of duplicating those (and making it more confusing by switching the method calls around)
We also generally do one line per argument if the list of arguments gets to long.
```cs
var alertLevelKey = state.StationAlert.Level != null ? $"alert-level-{state.StationAlert.Level}" : "comp-pda-ui-unknown";

StationAlertLevelLabel.SetMarkup(Loc.GetString(
    "comp-pda-ui-station-alert-level",
    ("ColorLevel", state.StationAlert.Color ?? "white"), 
    ("AlertLevel", Loc.GetString(alertLevelKey))
    ));

StationAlertLevelInstructions.SetMarkup(Loc.GetString(
    "comp-pda-ui-station-alert-level-instructions",
    ("AlertLevelInstructions", Loc.GetString($"{alertLevelKey}-instructions")))
    );
 ```